packages/components/src/internal/components/entities/EntityMoveModal.tsx

```diff
@@ -237,6 +250,7 @@ export const EntityMoveModal: FC<EntityMoveModalProps> = memo(props => {
                     title={title}
                     dataType={entityDataType.projectConfigurableDataType}
                     dataTypeRowId={dataTypeRowId}
+                    excludeCurrentAsTarget={maxSelected === 1}
```

Comment:
would maxSelected by 1 when you are moving a single sample from the sample overview page? just trying to understand the use-case for this excludeCurrentAsTarget

Reply:
Yes, that's the use case.